srcalc/src/test/java/gov/va/med/srcalc/vista/MockVistaPatientDao.java

```diff
@@ -10,7 +11,7 @@
     @Override
     public Patient getPatient(int dfn)
     {
-        return new Patient(dfn, "PATIENT,TEST" + dfn);
+    	return SampleObjects.dummyPatient();
```

Comment:
We're losing the provided DFN here. How about adding a DFN parameter to dummyPatient()?